content/docs/08-user-management/3-palette-resource-limits.md

```diff
@@ -36,6 +36,15 @@ import InfoBox from 'shared/components/InfoBox';
 The resource limit can be customized according to per tenant resource requirements 
 </InfoBox>
 
+# Palette API Rate Limits
+
+* API operations are limited to 10 requests per second for an IP Address.
+
+
+* The API request limits are categorized by resources such as  `/v1/cloudconfig/:uid` and `/v1/cloudconfig/:uid/`. Machine pools are counted for rate limits as both belong to the same cloud-config resource.
```

Comment:
@resmaa It should be like this:

The API request limits are categorized by resources such as `/v1/cloudconfig/:uid` and `/v1/cloudconfig/:uid/machinepools`. Both API requests are counted for the same rate limits as both belong to the same cluster's cloud config resource.

The API requests limits are categoried by resource like both the /v1/cloudconfig/:uid & /v1/cloudconfig/:uid/machinepools are counted for rate limit as both belongs to the same cloudconfig resource.